package/koreader/package

```diff
@@ -0,0 +1,26 @@
+# vim: set ft=sh:
+pkgname=koreader
+pkgdesc="KOReader"
+url=https://github.com/koreader/koreader
+pkgver=2020.08.01-1
+timestamp=2020-09-19T03:30Z
+section=utils
+maintainer="raisjn <of.raisjn@gmail.com>"
+license=AGPL3
+
+image=base
+source=(https://github.com/koreader/koreader/releases/download/v2020.08.1/koreader-remarkable-v2020.08.1.zip)
+sha256sums=(447c151e11bf56d71145cecc731dfa2a39eb026fcce4c4d0571cc8e431d5c491)
+
+build() {
+  # Using prebuilt binaries for koreader
+  return
+}
+
+package() {
+  install -d "$pkgdir"/opt/koreader
+  cp -R "$srcdir"/* "$pkgdir"/opt/koreader/
```

Comment:
For some reason, `$srcdir` also contains the zip (`koreader-remarkable-v2020.08.1.zip`) again (and gets packaged to `/opt/koreader`) which makes the package basically twice as large for no reason.

Reply:
Also should we really put standalone apps directly into `/opt` as if we didn't have another rootfs there? Alternativly I would suggest `/opt/lib` for that as many linux packages also put their additional data there when `(/opt)/usr/share/` is not fitting for all their content.

Reply:
Maybe `/opt/usr/local/koreader` makes more sense?

Reply:
where would you recommend putting standalone apps? 

good find on the .zip, should definitely be removed

Reply:
Under Archlinux's rules you'd put them in `/opt/{name}` but since we are already here it might make more sense to use `/usr/local/{name}`. That said, reviewing the guidelines here https://wiki.archlinux.org/index.php/Arch_package_guidelines#Directories leads me to believe that doesn't make sense either.

Reply:
`/opt/usr/local/koreader` is a really good idea, too. Though if I'm not mistaken, `/usr/local/bin/` usually means "No managed by the system package manager". Looking at my system that doesn't seem to hold much merit though.

So I think it doesn't really matter for us. I have no problem with `/opt/usr/local/...` for the package as well.

Reply:
we could do `/opt/usr/local/apps/koreader` or someting